fetch/metadata/redirect/multiple-redirect-https-downgrade-upgrade.sub.html

```diff
@@ -6,68 +6,13 @@
 <script src=/fetch/metadata/resources/redirectTestHelper.sub.js></script>
 <script src=/common/security-features/resources/common.sub.js></script>
 <script src=/common/utils.js></script>
-<style>
-  @font-face {
-    font-family: myDowngradeUpgradeFont;
-    src: url(https://{{host}}:{{ports[https][0]}}/fetch/api/resources/redirect.py?location=http%3A%2F%2F{{host}}%3A{{ports[http][0]}}%2Ffetch%2Fapi%2Fresources%2Fredirect.py%3Flocation%3Dhttps%253A%252F%252F{{host}}%253A{{ports[https][0]}}%252Ffetch%252Fmetadata%252Fresources%252Frecord-header.py%253Ffile%253Dfont-https-downgrade-upgrade{{$id:uuid()}});
-  }
-  #fontTest {
-    font-family: myDowngradeUpgradeFont;
-  }
-</style>
 <body>
-<div id="fontTest">Downgraded then upgraded font</div>
 <script>
   let nonce = "{{$id}}";
```

Comment:
This is actually broken, as the `{{$id:uuid()}}` call on 12 is gone. But also: we don't need this line anymore. :)

Reply:
The test needs `nonce` to be defined to run:

```
promise_test: Unhandled rejection with value: object "ReferenceError: nonce is not defined"
```

However, this was indeed broken. I've fixed it by using `token()` instead.